runbotics/runbotics-orchestrator-ui/src/src-app/views/process/EditProcessDialog/TagsInput/TagsInput.types.ts

```diff
@@ -0,0 +1,17 @@
+
+import { Tag } from 'runbotics-common';
+
+export interface TagsInputProps {
+    selected: TagName[]; // selectedTagsNames
```

Comment:
Maybe consider using those commented out names of functions and fields? It's up to you, but imho the longer names are more informative.

Reply:
good point, although the idea was to do not keyword 'tag' in 'TagsInput' component (ofc, the params are related to tags). In more general scope the params passed are better described:
![image](https://github.com/runbotics/runbotics/assets/69820069/8daad9c0-5793-4fc8-8cac-fc0ab190a205)

I needed the comments because I was moving the existing functionality to separate component. 

Can it be left as it is or this shouldn't be, what do you think?